platform/lf_nrf52_support.c

```diff
@@ -55,11 +59,11 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 static const nrfx_timer_t g_lf_timer_inst = NRFX_TIMER_INSTANCE(3);
 
 /**
- * Keep track of interrupts being raised.
- * Allow sleep to exit with nonzero return on interrupt.
+ * Flag to be raised when overflow occurs and is corrected.
  */
-bool _lf_timer_interrupted = false;
 bool _lf_overflow_corrected = false;
+
+// FIXME: what is this for?
```

Comment:
This is (was?) for passing to `sd_nvic_critical_region_enter` and `sd_nvic_critical_region_exit`.

Reply:
That I know, but I wasn't sure what purpose it serves :-) I think it's a counter to figure out the depth of nested critical sections...